clientgui/TermsOfUsePage.cpp

```diff
@@ -222,10 +224,19 @@ void CTermsOfUsePage::OnPageChanged( wxWizardExEvent& event ) {
         _("Please read the following terms of use:")
     );
 
-    m_pTermsOfUseCtrl->SetValue(
-        wxString(pc.terms_of_use.c_str(), wxConvUTF8)
-    );
-    m_pTermsOfUseCtrl->SetSelection(0,0);
+    std::string tou = pc.terms_of_use;
+    xml_unescape(tou);
```

Comment:
I think this should go to the code that reads XML. Any objects that comes out of XML parser should be usable without any further unescaping.

Reply:
This will make it harder to identify whether we got html tou or text

Reply:
Fixed